### PR TITLE
Feat/categories test filter

### PR DESCRIPTION
**Pull request by andresz1**
## Description
Execute test for some categories only with the studio using `runtime-mocha`

## Related Issue
None

## Example
```
sui-studio test --categories user,shipping
```

### CODE REVIEW

**Comment by midudev** [comment]
Instead an environment variable, this should be an option for the `test` command.

File: packages/sui-studio/README.md [hunk 1]
```diff
@@ -272,17 +272,15 @@ Add this scripts on your own components project
...
-	test:watch: "sui-studio test --watch"
-    ...
+	  "test": "sui-studio test",
+	  "test:watch": "sui-studio test --watch"
   }
-  ...
 }
 ```
 
+If you want to execute the tests for some specific categories only use `CATEGORIES` environment variable. It takes a comma separeted set of category names (e.g. `CATEGORIES="user,shipping" sui-studio test`)
```
Comment (jmanrumartinez) [inline packages/sui-studio/README.md:282]:
```suggestion
If you want to execute the tests for some specific categories only use `CATEGORIES` environment variable. It takes a comma separated set of category names (e.g. `CATEGORIES="user,shipping" sui-studio test`)
```

**Review by midudev — APPROVED** [review]
We will move this to an option for the CLI in the next major.

**Comment by andresz1** [comment]
> Instead an environment variable, this should be an option for the `test` command.

Agree! I will try to do it. The problem was that it should be available in `runtime-mocha`

**Review by javiauso — APPROVED** [review]

**Comment by andresz1** [comment]
> We will move this to an option for the CLI in the next major.

Changes made @midudev thanks!

File: packages/sui-studio/README.md [hunk 2]
```diff
@@ -272,17 +272,15 @@ Add this scripts on your own components project
...
-	test:watch: "sui-studio test --watch"
-    ...
+	  "test": "sui-studio test",
+	  "test:watch": "sui-studio test --watch"
   }
-  ...
 }
 ```
 
+Use `-C` or `--categories` parameter to execute tests for some categories only. It takes a comma separeted set of category names (e.g. `sui-studio test --categories user,shipping`)
```
Comment (midudev) [inline packages/sui-studio/README.md:282]:
```suggestion
Use `-C` or `--categories` parameter to execute tests for some categories only. It takes a comma separated set of category names (e.g. `sui-studio test --categories user,shipping`)
```

**Review by tomasmax — APPROVED** [review]

**Pull request merged**